lib/twilio/device.ts

```diff
@@ -1553,31 +1560,29 @@ namespace Device {
 
   /**
    * Emitted when the {@link Device} is unregistered.
-   * @param device
-   * @example `device.on('unregistered', device => { })`
+   * @example `device.on('unregistered', () => { })`
    * @event
    */
-  declare function unregisteredEvent(device: Device): void;
+  declare function unregisteredEvent(): void;
 
   /**
    * Emitted when the {@link Device} is registering.
-   * @param device
-   * @example `device.on('registering', device => { })`
+   * @example `device.on('registering', () => { })`
    * @event
    */
-  declare function registeringEvent(device: Device): void;
+  declare function registeringEvent(): void;
 
   /**
    * Emitted when the {@link Device} is registered.
-   * @param device
-   * @example `device.on('registered', device => { })`
+   * @example `device.on('registered', () => { })`
    * @event
    */
-  declare function registeredEvent(device: Device): void;
+  declare function registeredEvent(): void;
 
   /**
    * Emitted when the {@link Device}'s token is about to expire. Use DeviceOptions.refreshTokenMs
    * to set a custom warning time. Default is 10000 (10 seconds) prior to the token expiring.
+   * @param device
```

Comment:
Is this true? I don't see that we pass the device to the listener.

Reply:
Yes. See `this.emit('tokenWillExpire', this);` above. I think the example is wrong. I'll update.